src/NuGet.Services.EndToEnd/project.json

```diff
@@ -2,22 +2,17 @@
   "dependencies": {
     "Microsoft.AspNetCore.WebUtilities": "1.1.1",
     "Microsoft.Extensions.Configuration.Binder": "1.1.2",
```

Comment:
nit: can we move to `<PackageReference>`? Is should be pretty easy to just map all of these dependencies to `<PackageReference>` in the csproj.